pkg/views/project/create/view.go

```diff
@@ -1,21 +1,54 @@
 package create
 
 import (
+	"context"
 	"errors"
+	"fmt"
 
 	"github.com/charmbracelet/huh"
+	"github.com/goharbor/go-client/pkg/sdk/v2.0/client/registry"
+	"github.com/goharbor/harbor-cli/pkg/utils"
 	log "github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
 )
 
 type CreateView struct {
 	ProjectName  string
 	Public       bool
 	RegistryID   string
 	StorageLimit string
+	ProxyCache   bool
+}
+
+func getRegistryList() (*registry.ListRegistriesOK, error) {
```

Comment:
This will soon be gone; I'm centralizing it now. 
Note -> I am talking about the registryList() function.